index.bs

```diff
@@ -2338,23 +2338,23 @@ Methods</h4>
 		Let <code>buffer</code> be the {{AudioBuffer}} with
 		\(N_b\) frames, let \(N_f\) be the number of elements in the
 		{{AudioBuffer/copyFromChannel()/destination}} array, and \(k\) be the value of
-		{{AudioBuffer/copyFromChannel()/startInChannel}}. Then the number of frames copied
+		{{AudioBuffer/copyFromChannel()/bufferOffset}}. Then the number of frames copied
 		from <code>buffer</code> to {{AudioBuffer/copyFromChannel()/destination}} is
-		\(\min(N_b - k, N_f)\). If this is less than \(N_f\), then the
+		\(\max(0, \min(N_b - k, N_f))\). If this is less than \(N_f\), then the
 		remaining elements of {{AudioBuffer/copyFromChannel()/destination}} are not
 		modified.
 
 		<pre class=argumentdef for="AudioBuffer/copyFromChannel()">
 		destination: The array the channel data will be copied to.
 		channelNumber: The index of the channel to copy the data from. If <code>channelNumber</code> is greater or equal than the number of channels of the {{AudioBuffer}}, <span class="synchronous">an {{IndexSizeError}} MUST be thrown</span>.
-		startInChannel: An optional offset to copy the data from. If <code>startInChannel</code> is greater than the <code>length</code> of the {{AudioBuffer}}, <span class="synchronous">an {{IndexSizeError}} MUST be thrown</span>.
+		bufferOffset: An optional offset in to the {{AudioBuffer}} to copy the data from.
```

Comment:
`offset in to`? Not sure what this means.

Reply:
Fixed